File: persistence/src/main/java/pl/srw/billcalculator/pojo/ITauronPrices.java
package pl.srw.billcalculator.pojo;

import pl.srw.billcalculator.db.Prices;

public interface ITauronPrices extends Prices {

    String getEnergiaElektrycznaCzynna();
    String getOplataDystrybucyjnaZmienna();
    String getOplataDystrybucyjnaStala();
    String getOplataPrzejsciowa();
    String getOplataAbonamentowa();

    String getOplataDystrybucyjnaZmiennaNoc();
    String getOplataDystrybucyjnaZmiennaDzien();
    String getEnergiaElektrycznaCzynnaNoc();
    String getEnergiaElektrycznaCzynnaDzien();
}

File: persistence/src/main/java/pl/srw/billcalculator/pojo/ITauronPrices.kt
package pl.srw.billcalculator.pojo

import pl.srw.billcalculator.db.Prices

interface ITauronPrices : Prices {

    val energiaElektrycznaCzynna: String
    val oplataDystrybucyjnaZmienna: String
    val oplataDystrybucyjnaStala: String
    val oplataPrzejsciowa: String
    val oplataAbonamentowa: String
    val oplataOze: String

    val oplataDystrybucyjnaZmiennaNoc: String
    val oplataDystrybucyjnaZmiennaDzien: String
    val energiaElektrycznaCzynnaNoc: String
    val energiaElektrycznaCzynnaDzien: String
}
